src/types.ts

```diff
@@ -108,6 +108,20 @@ export type SyncTeXRecordToTeX = {
     column: number
 }
 
+export type SyncTeXRecordToPDFAll = {
+    Page: number;
+    x: number;
+    y: number;
+    h: number;
+    v: number;
+    W: number;
+    H: number;
+}
+
+export type SyncTeXRecordToPDFAllList = {
+    records: SyncTeXRecordToPDFAll[];
+}
+
```

Comment:
I don't think we need to explicitly define this type. What are the cases when `SyncTeXRecordToPDFAll[]` won't work?